packages/sn-filter-pane/src/hooks/use-render.ts

```diff
@@ -27,5 +31,5 @@ export default function useRender(stores: IStores) {
     return (() => {
       teardown(root);
     });
-  }, [resourcesReady]);
+  }, [resourcesReady, componentsHash]);
```

Comment:
did you try to only listen on fpLayout here instead of doing the hash thing? Not entirely sure how it works with the store etc. but the layout object could be a new object on every change and then it would work...

Reply:
In summary: `useStaleLayout` does not work – selections are still triggering a re-render
`componentsHash` has now been moved under flag too for safety so we can do more testing once the entire feature is ready.